Common subdirectories: lilo-22.8.ORIG/diagnose and lilo-22.8/diagnose
Common subdirectories: lilo-22.8.ORIG/doc and lilo-22.8/doc
Common subdirectories: lilo-22.8.ORIG/dos and lilo-22.8/dos
diff -u lilo-22.8.ORIG/Makefile lilo-22.8/Makefile
--- lilo-22.8.ORIG/Makefile	2007-03-20 10:15:59.000000000 +0000
+++ lilo-22.8/Makefile	2007-03-20 10:22:19.000000000 +0000
@@ -69,18 +69,7 @@
    -DMDPRAID -DDEVMAPPER
    
 
-# set the following if you wish LILO.COM to be installed
-DOS_DIR=/dosC/boot
-#
-# select one of the following to place the man pages in the right place
-#MAN_DIR=`if [ -z "$(whereis lilo | grep 'lilo.8')" ]; then \
-#	whereis lilo | sed -e "sX.* /usrX/usrX" -e "sX/man8/lilo.8.*XX" ; \
-#	elif [ -f /usr/bin/manpath ]; then \
-#  manpath | sed "s/:.*//"; else echo /usr/man; fi`
-#
-MAN_DIR=`if [ -f /usr/bin/manpath ]; then \
-	manpath | sed "s/:.*//"; else echo /usr/share/man; fi`
-#MAN_DIR=/usr/share/man
+MAN_DIR=/usr/share/man
 
 # set the compiler optimization level
 
@@ -165,10 +154,7 @@
 #
 # everything needed to run, just short of installation
 #
-all:	lilo bootsect.b diag1.img # OBSOLETE:  $(BOOTS) edit dparam.com activate
-	if [ -x /usr/bin/bcc -o -x /usr/local/bin/bcc ]; then \
-		make diagnostic; \
-		make dosexe; fi
+all:	lilo
 
 #
 # everything above plus the statically linked version
@@ -452,7 +438,6 @@
 	  rm -f $$ROOT$(BOOT_DIR)/boot.b; fi
 	cp mkrescue $$ROOT$(SBIN_DIR)
 	cp lilo $$ROOT$(SBIN_DIR)
-	strip $$ROOT$(SBIN_DIR)/lilo
 	cp keytab-lilo.pl $$ROOT$(USRSBIN_DIR)
 	cp manPages/lilo.8 $$ROOT$(MAN_DIR)/man8
 	cp manPages/mkrescue.8 $$ROOT$(MAN_DIR)/man8
Only in lilo-22.8: Makefile.orig
Common subdirectories: lilo-22.8.ORIG/manPages and lilo-22.8/manPages
Common subdirectories: lilo-22.8.ORIG/sample and lilo-22.8/sample
